### PR TITLE
Added i18n french support

### PR DESCRIPTION
**Pull request by csachdeva83**
- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Feature
- **What is the current behavior?** (You can also link to an open issue here)
https://github.com/novuhq/novu/issues/942
- **What is the new behavior (if this is a feature change)?**
Added i18n french support
- **Other information**:

### CODE REVIEW

**Comment by scopsy** [comment]
Hi, @csachdeva83 are you a native french speaker? Just so I know if native speakers should review this

**Comment by csachdeva83** [comment]
@scopsy I am not a native french speaker.

File: packages/notification-center/src/i18n/languages/fr.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { ITranslationEntry } from '../lang';
+
+export const FR: ITranslationEntry = {
+  translations: {
+    notifications: 'Avis',
+    markAllAsRead: 'tout marquer comme lu',
+    poweredBy: 'Alimenté par',
```
Comment (fredsossa1) [inline packages/notification-center/src/i18n/languages/fr.ts:7]: This works as a good translation but 'Propulsé par' sounds better.
```suggestion
    poweredBy: 'Propulsé par',
```

File: packages/notification-center/src/i18n/languages/fr.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { ITranslationEntry } from '../lang';
+
+export const FR: ITranslationEntry = {
+  translations: {
+    notifications: 'Avis',
```
Comment (fredsossa1) [inline packages/notification-center/src/i18n/languages/fr.ts:5]: In french we also just use 'Notification'
```suggestion
    notifications: 'Notifications',
```

File: packages/notification-center/src/i18n/languages/fr.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+    markAllAsRead: 'tout marquer comme lu',
```
Comment (fredsossa1) [inline packages/notification-center/src/i18n/languages/fr.ts:6]: Honestly this is just a small detail
```suggestion
    markAllAsRead: 'Tout marquer comme lu',
```

**Comment by scopsy** [comment]
Thank you all! Merging this 🙏

**Review by scopsy — APPROVED** [review]

**Pull request merged**